src/product/tests/product.mock.ts

```diff
@@ -0,0 +1,41 @@
+import { v4 as uuid } from 'uuid';
+import { faker } from '@faker-js/faker';
+
+import { PaginationDto } from '@src/common/dto/pagination.dto';
+import { Sorting } from '@src/common/types/pagination';
+import { CreateProductDto, UpdateProductDto } from '@product/dto';
+import { ProductState } from '@product/types/product';
+import { Product } from '@product/entities/product.entity';
+
+export const mockProduct: Product = {
+  id: uuid(),
+  name: faker.commerce.productName(),
+  description: faker.commerce.productDescription(),
+  state: ProductState.N,
```

Comment:
There is available a faker helper to use random values of your enum `faker.helpers.enumValue(ProductState.N)` if that's the case of use

Reply:
thanks